docs/changelog.rst

```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* :release:`1.2.2 <2020-03-xx>` 783
```

Comment:
Don't add a release above it. Just leave it empty as we don't know if it will be minor/patch or the date.